sovtoken/sovtoken/test/test_public_mint.py

```diff
@@ -337,3 +344,90 @@ def assert_valid_minting(helpers, amount):
+                                  dest=new_did,
+                                  verkey=verkey)
+        helpers.sdk.send_request_objects([req], wallet=sdk_wallet_trustee)
+        looper.run(eventually(
+            lambda: assertExp(
+                get_uncommitted_txns_count_for_pool(nodeSetWithIntegratedTokenPlugin,
+                                                    DOMAIN_LEDGER_ID) == uncommitted_size_before_domain + 1
+            )))
+        looper.run(eventually(
+            lambda: assertExp(
+                get_committed_txns_count_for_pool(nodeSetWithIntegratedTokenPlugin,
+                                                  DOMAIN_LEDGER_ID) == committed_size_before_domain
+            )))
+
+        # Do mint with a newly added Trustee
+        [address1, *_] = addresses
+        outputs = [{ADDRESS: address1, AMOUNT: 100}]
+        request = helpers.request.mint(outputs, number_signers=2)
+        request = looper.loop.run_until_complete(
+            multi_sign_request(sdk_wallet_trustee[0], new_did, json.dumps(request.as_dict)))
+
+        request = json.loads(request)
+        sigs = request["signatures"]
+        request = helpers.sdk.sdk_json_to_request_object(request)
+        setattr(request, "signatures", sigs)
+        helpers.sdk.send_request_objects([request])
+
+        # make sure we have both txns uncommitted
+        looper.run(eventually(
+            lambda: assertExp(
+                get_uncommitted_txns_count_for_pool(nodeSetWithIntegratedTokenPlugin,
+                                                    DOMAIN_LEDGER_ID) == uncommitted_size_before_domain + 1
+            )))
+        looper.run(eventually(
+            lambda: assertExp(
+                get_committed_txns_count_for_pool(nodeSetWithIntegratedTokenPlugin,
+                                                  DOMAIN_LEDGER_ID) == committed_size_before_domain
+            )))
+        looper.run(eventually(
+            lambda: assertExp(
+                get_uncommitted_txns_count_for_pool(nodeSetWithIntegratedTokenPlugin,
+                                                    TOKEN_LEDGER_ID) == uncommitted_size_before_token + 1
+            )))
+        looper.run(eventually(
+            lambda: assertExp(
+                get_committed_txns_count_for_pool(nodeSetWithIntegratedTokenPlugin,
+                                                  TOKEN_LEDGER_ID) == committed_size_before_token
+            )))
+
+    # make sure we have both txns committed
+    looper.run(eventually(
+        lambda: assertExp(
+            get_committed_txns_count_for_pool(nodeSetWithIntegratedTokenPlugin,
+                                              DOMAIN_LEDGER_ID) == committed_size_before_domain + 1
+        )))
+    looper.run(eventually(
+        lambda: assertExp(
+            get_committed_txns_count_for_pool(nodeSetWithIntegratedTokenPlugin,
+                                              TOKEN_LEDGER_ID) == committed_size_before_token + 1
```

Comment:
Some time ago when debugging tests which used `assertExp` I found that pytest doesn't output any useful information except that assert failed. For that reason I've created another helper called `assert_eq` which accepts two parameters (left-hand and right-hand side) and in case of assertion failure outputs their values, which often can lead to faster problem detection. So, I recommend using this helper here as well.